docs/metrics.md

```diff
@@ -282,6 +282,9 @@ Returns the amount of space in bytes restored from the source virtual machine. T
 ### kubevirt_vmsnapshot_persistentvolumeclaim_labels
 Returns the labels of the persistent volume claims that are used for restoring virtual machines. Type: Gauge.
 
+### kubevirt_vmsnapshot_succeeded_timestamp_seconds
+Virtual Machine succeeded snapshots timestamp. Type: Gauge.
```

Comment:
@avlitman 
Virtual Machine succeeded snapshots timestamp -> Returns the timestamp of successful virtual machine snapshot.

Reply:
fixed